src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -202,7 +205,8 @@ public SauceOnDemandBuildWrapper(
             boolean launchSauceConnectOnSlave,
             boolean useOldSauceConnect,
             boolean verboseLogging,
-            boolean useLatestVersion
+            boolean useLatestVersion,
+            boolean useGeneratedTunnelIdentifier
```

Comment:
is there any harm in always using the generated Tunnel Identifier rather than adding a Boolean configuration option?